transform/mattermost-analytics/models/intermediate/hightouch/int_customers_with_onprem_subs.sql

```diff
@@ -0,0 +1,188 @@
+{{ config
```

Comment:
Not sure why we this model is needed. Stripe model should be enough. As soon as the final table is created, then it can be compared with the table from the old project.

Reply:
You are right, I dropped this model and re-wrote the code using Stripe.